config/config.example.yaml

```diff
@@ -8,3 +8,10 @@ slack:
   # if enabled, siera-kube-watch will post a message through slack url
   enabled: false
   url: "http://127.0.0.1/slack"
+
+# In case you want to filter your event stream from specific reason, you can add that reason to excluded.reasons.
+# excluded.reasons is an array of reasons that will be excluded so any event that have one of these reasons won't be
+# published.
+# Example:
+#   excluded.reasons: ["FailedGetResourceMetric", "ScalingReplicaSet", "Started", "Killing", "Unhealthy"]
```

Comment:
exclude reasons only come from type normal right?

Reply:
from all type actually. Since I just include event with reason SCALING_REPLICA_SET, STARTED, or KILLING from event with type normal (https://github.com/warungpintar/siera-kube-watch/blame/master/util/helper.go#L15), mostly this configuration will exclude event from type warning except for 3 reasons above.

Reply:
got it